test_conformance/basic/main.c

```diff
@@ -136,27 +136,27 @@ test_definition test_list[] = {
     ADD_TEST( kernel_memory_alignment_constant ),
     ADD_TEST( kernel_memory_alignment_private ),
 
-    ADD_TEST( progvar_prog_scope_misc ),
-    ADD_TEST( progvar_prog_scope_uninit ),
-    ADD_TEST( progvar_prog_scope_init ),
-    ADD_TEST( progvar_func_scope ),
+    ADD_TEST_VERSION( progvar_prog_scope_misc, Version(2, 0) ),
+    ADD_TEST_VERSION( progvar_prog_scope_uninit, Version(2, 0) ),
+    ADD_TEST_VERSION( progvar_prog_scope_init, Version(2, 0) ),
+    ADD_TEST_VERSION( progvar_func_scope, Version(2, 0) ),
 
     ADD_TEST( global_work_offsets ),
     ADD_TEST( get_global_offset ),
 
-    ADD_TEST( global_linear_id ),
-    ADD_TEST( local_linear_id ),
-    ADD_TEST( enqueued_local_size ),
+    ADD_TEST_VERSION( global_linear_id, Version(2, 0) ),
+    ADD_TEST_VERSION( local_linear_id, Version(2, 0) ),
+    ADD_TEST_VERSION( enqueued_local_size, Version(2, 0) ),
 
-    ADD_TEST( simple_read_image_pitch ),
-    ADD_TEST( simple_write_image_pitch ),
+    ADD_TEST_VERSION( simple_read_image_pitch, Version(2, 0) ),
+    ADD_TEST_VERSION( simple_write_image_pitch, Version(2, 0) ),
```

Comment:
Why do the two `{read,write}_image_pitch` require CL2? I couldn't spot anything obvious skimming the code.

Reply:
My assumption was that this test was not present in cl12_trunk test suite. But you are right there is no reason why it should not be enabled. Moreover we have this test on internal branch enabled for CL1.2.